conf/local.conf

```diff
@@ -81,3 +81,10 @@ UBOOT_SIGN_KEYNAME ?= "ubootdev"
 # custom signing key just generate a custom RSA 2048 key (PEM format) and
 # set via the OPTEE_TA_SIGN_KEY variable.
 OPTEE_TA_SIGN_KEY ??= "${TOPDIR}/conf/keys/opteedev.key"
+
+#
+# UEFI Secure Boot
+#
+# Folder for UEFI keys and certificates
+UEFI_SIGN_KEYDIR ??= "${TOPDIR}/conf/keys/uefi"
+#UEFI_SIGN_ENABLE ?= "1"
```

Comment:
To validate the PR  https://github.com/foundriesio/meta-lmp/pull/682 we need this bit enabled or not?

Reply:
It is enabled for intel only (at meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc):

```
# Intel
UEFI_SIGN_ENABLE:intel-corei7-64 = "1"
```

Reply:
right